src/Main.idr

```diff
@@ -39,5 +40,5 @@ main = eitherT putStrLn pure $ do
           $ consumes' [JSON]
               { a = Example }
               (\ctx => text "Content cannot be parsed: \{ctx.request.body}" ctx >>= status BAD_REQUEST)
-          $ \ctx => text (show ctx.request.body) ctx >>= status OK
+          $ \ctx => json (encode ctx.request.body) ctx >>= status OK
```

Comment:
I guess the `encode` is no longer needed for the example.

Reply:
Also good spot. Updated 👍